### PR TITLE
Release 11.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+
+**Fixed bugs:**
+
+- Fix `grafana_user` and switch to using `flush` [\#283](https://github.com/voxpupuli/puppet-grafana/pull/283) ([alexjfisher](https://github.com/alexjfisher))
+- Set grafana.ini owner to root with packages [\#264](https://github.com/voxpupuli/puppet-grafana/pull/264) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Switch documentation to puppet strings [\#294](https://github.com/voxpupuli/puppet-grafana/pull/294) ([SimonHoenscheid](https://github.com/SimonHoenscheid))
+- Disable Arch Linux testing [\#292](https://github.com/voxpupuli/puppet-grafana/pull/292) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (ekohl) [inline CHANGELOG.md:31]: Since it's a major release anyway, should this be listed as a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
...
+- Accept `Sensitive` `cfg` [\#280](https://github.com/voxpupuli/puppet-grafana/pull/280) ([alexjfisher](https://github.com/alexjfisher))
+
+**Fixed bugs:**
+
+- Fix `grafana_user` and switch to using `flush` [\#283](https://github.com/voxpupuli/puppet-grafana/pull/283) ([alexjfisher](https://github.com/alexjfisher))
+- Set grafana.ini owner to root with packages [\#264](https://github.com/voxpupuli/puppet-grafana/pull/264) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Switch documentation to puppet strings [\#294](https://github.com/voxpupuli/puppet-grafana/pull/294) ([SimonHoenscheid](https://github.com/SimonHoenscheid))
```
Comment (ekohl) [inline CHANGELOG.md:30]: Enhancement?

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: updated it

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md]: updated it

**Review by ekohl — APPROVED** [review]

**Pull request merged**